beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -63,6 +63,11 @@ func (r *RegularSync) validateBeaconAttestation(ctx context.Context, msg proto.M
 	if fromSelf {
 		return false, nil
 	}
+	finalizedEpoch := r.chain.FinalizedCheckpt().Epoch
+	if finalizedEpoch >= att.Data.Source.Epoch || finalizedEpoch >= att.Data.Target.Epoch {
+		log.Debugf("Rejecting old attestation with root %#x and target epoch of %d ", attRoot, att.Data.Target.Epoch)
```

Comment:
what about log for `finalizedEpoch >= att.Data.Source.Epoch` case?

Reply:
added it in as a field for source epochs